chia/full_node/coin_store.py

```diff
@@ -82,7 +82,7 @@ async def new_block(
         height: uint32,
         timestamp: uint64,
         included_reward_coins: Collection[Coin],
-        tx_additions: Collection[Coin],
+        tx_additions: Collection[tuple[bytes32, Coin]],
```

Comment:
in this PR, I just made additions be `[Coin, Bytes32]`. I would think it would be simpler to stay compatible with that.
https://github.com/Chia-Network/chia_rs/pull/1127

Reply:
Hmm even if this is related to `ForkInfo` and not `additions_and_removals`? https://github.com/Chia-Network/chia-blockchain/pull/19749/files#diff-40409ea13958d9e8cec2be388c12e68da7c3179de55f01879713c5abdb1fed38R567-R568

Reply:
we use `additions_and_removals()` to update `ForkInfo`, don't we?
even if we don't, I would think it's easier to be consistent.
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/consensus/blockchain.py#L277-L284

Reply:
We're constructing these tuples from `ForkInfo`'s `additions_since_fork` map, but I like the consistency point so I'll adopt that.